FakeNFT/Services/Requests/NFTRequest.swift

```diff
@@ -0,0 +1,27 @@
+//
+//  NFTRequest.swift
+//  FakeNFT
+//
+//  Created by D on 05.09.2023.
+//
+
+import Foundation
+
+struct NFTRequest: NetworkRequest {
+    let id: Int
+
+    init(id: Int) {
+        self.id = id
+    }
+    var endpoint: URL? {
+        get {
+            URL(string: self.baseEndpoint + "nft/\(id)")
```

Comment:
Можно использовать интерполяцию для эндпоинтов
```suggestion
            URL(string: "\(baseEndpoint)nft/\(id)")
```